pkg/addons/default/assets/aws-node.yaml

```diff
@@ -64,12 +64,13 @@ spec:
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
+      priorityClassName: system-node-critical
       serviceAccountName: aws-node
       hostNetwork: true
       tolerations:
       - operator: Exists
       containers:
-      - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.3.2
+      - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.4.0
```

Comment:
```suggestion
      - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.4.1
```

Reply:
This one is actually generated, so I need to update the URLs here:

https://github.com/weaveworks/eksctl/blob/312ba0cef113a6f98d1e0743eb0df161aebcdf98/pkg/addons/default/generate.go#L3-L4

And run `make generate`.